actions/benchmarkActions.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'isomorphic-fetch';
+import { API_HOST } from './../actions';
+
+const url = '/api/benchmarks';
+
+const START = 'yasp/benchmark/START';
+const DONE = 'yasp/benchmark/DONE';
+const ERROR = 'yasp/benchmark/ERROR';
+
```

Comment:
I'd rather use similar naming schemes as compared to the other async actions (I think it's REQUEST, OK, and ERROR), but I don't think it's a big deal if you don't feel like refactoring that.